features/storage/FEATURE_STORAGE/cfstore/configuration-store/configuration_store.h

```diff
@@ -873,6 +875,7 @@ typedef struct _ARM_DRIVER_CFSTORE
 
 
 
+MBED_DEPRECATED_SINCE("mbed-os-5.5", "CFSTORE replace by the fat filesystem driver.")
```

Comment:
From comments I field on developer.mbed.org we get criticized for our vague messages when deprecated is used. In this case, not sure what a filesystem driver is or where to look for one. Can we add some specific actions such as use: ```class obj```